config/locales/en/controllers/account_links.yml

```diff
@@ -0,0 +1,15 @@
+---
+en:
+  account_links:
+    controller:
```

Comment:
I removed the controller part of the keys so we can have relative translation keys in the controllers too. I would still keep controller and view translations separated though because I think it makes sense to make that division in model/controller/view translations. (also leads to smaller translation files)